### PR TITLE
Add * to comments from staff on preview

### PR DESCRIPTION
**Pull request by paulkaplan**
This PR will show the staff * next to staff usernames on comments, once that information starts coming from the API on comments.

### CODE REVIEW

File: src/views/preview/comment/comment.jsx [hunk 1]
```diff
@@ -214,6 +216,7 @@ Comment.propTypes = {
     author: PropTypes.shape({
         id: PropTypes.number,
         image: PropTypes.string,
+        is_staff: PropTypes.bool,
```
Comment (rschamp) [inline src/views/preview/comment/comment.jsx]: Even though this is what we use, we probably would want to clean up this name in the API so it was just `isScratchTeam` or `scratchteam`?

Comment (paulkaplan) [reply src/views/preview/comment/comment.jsx]: hmm yeah, although as with `isPublished` vs. `is_published`, I think it gets even more confusing when you start renaming properties...

Comment (rschamp) [reply src/views/preview/comment/comment.jsx]: Kinda... this is the job of our models in the API though. There are plenty of other places where we don't blindly pass though database fields.

Comment (paulkaplan) [reply src/views/preview/comment/comment.jsx]: Ok, well let me know what it should be and I'll change it here and in my PR against API https://github.com/LLK/scratch-api/pull/614

**Comment by paulkaplan** [comment]
@rschamp updated

**Pull request merged**